swift/swift.py

```diff
@@ -220,13 +220,31 @@ def _swiftcall(self, msg: str):
                 print("The message was ignored because "
                       "args of the create action have no such key; name or info.")
                 return
-            self.createApp(args["name"], AppInfo(**args["info"]))
+            name, info = args["name"], args["info"]
+            reply = QMessageBox.warning(
+                None,
+                "swift-call",
+                f"The app {self.sender().name} requests for creating an app {name}",

```

Comment:
I think the message should contain `info` in order for the user to determine it.

Reply:
However, maybe my next PR would resolve this because I am going to generalize the swiftcall mechanism.